tests/main/fake-netplan-apply/task.yaml

```diff
@@ -38,7 +38,7 @@ prepare: |
         sed "$TESTSLIB/snaps/netplan-snap/meta/snap.yaml.in" -e "s/base: BASESNAP/base: core20/" > "$TESTSLIB/snaps/netplan-snap/meta/snap.yaml"
         snap pack "$TESTSLIB/snaps/netplan-snap" --filename=netplan-snap-20.snap
         snap install --dangerous netplan-snap-20.snap
-    elif os.query is-jammy; then
+    elif os.query is-jammy || os.query is-ubuntu 22.10; then
```

Comment:
You forgot to update the first part of the `if` condition. :-)

Reply:
updated